common.py

```diff
@@ -3,7 +3,7 @@
 from fnmatch import fnmatch
 from glob import glob
 from logging import debug, info, error
-from os import path
+from os import path, environ
```

Comment:
Please use setvar(cc="...", cxx="...") in [toolchain-m68k:243](https://github.com/cahirwpz/amigaos-cross-toolchain/blob/master/toolchain-m68k#L243) and `fill_in` function. Does exactly the same but without environment lookup.